Backend/stack.yaml

```diff
@@ -79,7 +79,7 @@ extra-deps:
     subdirs:
       - app/gateway
       - app/mock-registry
-  -  ../../shared-kernel/lib/mobility-core
+  # -  ../../shared-kernel/lib/mobility-core
```

Comment:
Why comment? We don't need this, it should be removed completely

Reply:
> Why comment? We don't need this, it should be removed completely

We need it to run changed kernel code without pushing it (and changing commit link, downloading it, compiling it etc.). Personally i use it pretty often, that's why it is commented here.